backend/src/hatchling/bridge/app.py

```diff
@@ -127,19 +130,19 @@ def __init__(self, app):
         self.display_mini_header = app.display_mini_header
 
 
-def format_app_command(method, *args, **kwargs):
+def format_app_command(method: str, *args: tuple[Any, ...] | None, **kwargs: dict[str, Any | None]) -> str:
```

Comment:
why `dict[str, Any | None]` rather than `dict[str, Any]`?

Reply:
It should have been `dict[str, Any] | None`, but I also realise it isn't needed, so I am removing the union.